### DISCUSSION TITLE
Will the battery be bypassed to run only from power adapter when limit is reached (Non-pro Al Dente)

### DISCUSSION THREAD

**Discussion by MortMort** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#488] [Q&A]
Like the title says: Will the battery be bypassed such that the MacBook draws power solely from the power adapter when limit is reached? My thought is that this should surely decrease wear on the battery.

I'm asking because my laptop is barely 2 months old, has 34 battery cycles and already a maximum capacity of 97%. I think it's a lot of lost maximum capacity with few cycles inside a short timeframe.

 (Non-pro Al Dente version)

**Comment by MatthiasKerbl** [comment]
Hi @MortMort,

In short: Yes, when you use the Charge Limiter of AlDente your MacBook will primarily use the power brick as a power source. However, some discharges and charges can still accure. AlDente Pro has a feature called Sailing Mode to reduce this even further, you can read more about it in our blog: https://apphousekitchen.com/feature-explanation-sailing-mode/

Cheers,
Matthias
